### ISSUE TITLE
[BUG] no-op with runSQL migrations

### ISSUE THREAD

**Issue by nxmai** [jayvynl/django-clickhouse-backend#91]
**Describe the bug**
I use migrations.runSQL() to perform some migrations that the ORM cannot support for clickhouse DB. And it not work. 
The migrations code in clickhouse_backend/patch/migrations.py does not include special migrations of Django (from django.db.migrations.operations.special) 
Is it a bug or you guys dont want to perform that action?

```python
# Generated by Django 3.2.14 on 2024-07-11 12:49

from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0001'),
    ]

    operations = [
        migrations.RunSQL(
            'ALTER TABLE "event"  ADD COLUMN "record" String CODEC(ZSTD(6))'
        )
    ]

```
**Versions**

- ClickHouse server version. Version can be obtained by running `SELECT version()` query.
- Python version.
- Clickhouse-driver version.
- Django version.
- Django clickhouse backend version.

**Comment by jayvynl** [comment]
duplicate #15

**Comment by nxmai** [comment]
Do you have any way to decrease the memory, it use a lot of mem, I try to
set connection pool lower but not affect

**Comment by nxmai** [comment]
one more question, when will you update the clickhouse-driver version to
0.2.8
I found the issue is because the date/date32 field in taking a lot of
memory. cre:
https://clickhouse-driver.readthedocs.io/en/latest/types.html#date-date32

**Comment by jayvynl** [comment]
released as v1.3.0
